Dockerfile

```diff
@@ -1,12 +1,12 @@
 ARG PLATFORM=""
 ARG APP_INSIGHTS_AGENT_VERSION=3.4.13
```

Comment:
Please update this to at least 3.5.4, though I suggest to the latest which is 3.6.2

Reply:
thank you thank you